### ISSUE TITLE
No visible focus on Operating Status alert on VAMC Location pages

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#11699]
## Describe the defect
Within the VAMC locations pages, when a location has an operating status other than normal an alert displays. This alert is a link to the operating status page and as an interactive element should have a visible gold focus halo around the element when a user tabs to it. However, this is currently not available and should be updated.

## To Reproduce
1. Go to any VAMC locations page with an active operating status, example [Bessemer VA Clinic](https://www.va.gov/birmingham-health-care/locations/bessemer-va-clinic/)
2. Use the keyboard and the "Tab" key to navigate down the page
3. Notice that on the "On this page" links, when you are focusing on those, there is a gold halo outline around the links
4. Conversely, notice that when you are focused on the operating status alert, there is no visible gold halo

## AC / Expected behavior
- [ ] A gold halo, identical to the existing focus state on other interactive elements, should be visible when the operating status link receives focus
- [ ] Accessibility Lead review

## Screenshots
**Example of focus halo on "On this page" link:**
![image](https://user-images.githubusercontent.com/106678594/203613272-00711322-ab5c-4377-87a1-5f88f83f7688.png)
**Example of missing focus state (it's hard to show something not there but I swear I am focusing on the Operating status):**
![image](https://user-images.githubusercontent.com/106678594/203617415-51ba9380-3c0e-4ec5-8d53-10249ea08090.png)

 ## Additional context
I did this locally with the following updates. Obviously it doesn't have to be done in this exact way but I thought it might help:

- Adding `display:block` to `a.operating-status-link`
![image](https://user-images.githubusercontent.com/106678594/203618335-a7a7792d-7d85-4410-90e0-fce6a9ee2d66.png)
- Removing `height: 40px` from `.operating-status-flag`
![image](https://user-images.githubusercontent.com/106678594/203618299-1a9f6fab-2ba8-4688-92f5-a8e92962ceea.png)
- Updating the attribute on the <span> within the link:
  - `<span class="operating-status-flag operating-status-flag-warning operating-status usa-alert usa-alert-warning background-color-only vads-u-margin-top--1 vads-u-padding-y--2 vads-u-padding-x--1p5">`
  - Change `vads-u-margin-top--1` to `vads-u-margin-top--0`

## Timing/Priority
This does not need to be injected into the current sprint.  However, this should be completed within the next 1-2 sprints. Possible consideration for completion with the other work that will be done on the COVID alert banners, if it makes sense to complete these together.

### CMS Team
Please check the team(s) that will do this work.

- [ ] `Program`
- [ ] `Platform CMS Team`
- [ ] `Sitewide Crew`
- [ ] `⭐️ Sitewide CMS`
- [ ] `⭐️ Public Websites`
- [x] `⭐️ Facilities`
- [ ] `⭐️ User support`

**Comment by dsinla** [comment]
@maxx1128 Assigning a "2", please let me know if that works for you

**Comment by maxx1128** [comment]
@dsinla I haven't been able to check this locally since none of the locations in the description's link have a status alert (it may have been updated since I checked). But I agree this is likely a 2.

**Comment by dsinla** [comment]
@maxx1128 @dawnpruitt @davidmpickett Adding to Sprint 73 
(Interesting that I don't see in the history when this ticket was removed)

**Comment by laflannery** [comment]
This white space should really not be here, the focus outline should look the same as all other focus rings:
![image](https://user-images.githubusercontent.com/106678594/208522002-9973f279-8048-4485-8546-cd9f8e43259c.png)

Is there a way this can get updated using the update I mentioned originally in the description:

> Updating the attribute on the <span> within the link:
>   `<span class="operating-status-flag operating-status-flag-warning operating-status usa-alert usa-alert-warning background-color-only vads-u-margin-top--1 vads-u-padding-y--2 vads-u-padding-x--1p5">`
>   Change `vads-u-margin-top--1` to `vads-u-margin-top--0`

Please let me know if I should open a new ticket

**Comment by dawnpruitt** [comment]
@davidmpickett @JQuaggles
